client/sites-dashboard/components/sites-site-launch-nag.tsx

```diff
@@ -71,13 +71,27 @@ const getChecklistSlug = ( site: SiteExcerptData ) => {
 		return 'legacy-site-setup';
 	}
 
+	if ( intent === 'assembler-first' ) {
+		return 'legacy-site-setup';
+	}
+
 	const isHostedSite =
 		'host-site' === intent || 'new-hosted-site' === flow || 'import-hosted-site' === flow;
 
 	if ( isHostedSite && ! site?.is_wpcom_atomic ) {
 		return 'legacy-site-setup';
 	}
 
+	if ( site.launch_status === 'launched' ) {
+		if ( intent === 'build' ) {
+			return 'intent-build';
+		}
+
+		if ( intent === 'write' ) {
+			return 'intent-write';
+		}
+	}
```

Comment:
@p-jackson I needed these for matching state between /sites and /home for new sites that complete the first task list shown.

The build => intent-build progression was standard for "skip to dashboard" on the goals step. This code is matching: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Subzr%2Sivrjf.cuc%3Se%3Qn3r5p0qo%23128-og

The write => intent-write progression can be checked by slecting publish a blog goal in the goals step.